Content.Client/Verbs/UI/VerbMenuPresenter.cs

```diff
@@ -196,7 +202,26 @@ public override void OnKeyBindDown(ContextMenuElement element, GUIBoundKeyEventA
                     return;
             }
 
-            _verbSystem.ExecuteVerb(CurrentTarget, verb, verbElement.Type);
+            if (verb.ConfirmationPopup)
```

Comment:
Not directly related to this PR, but there's a comment above this that was just never updating during the last PR that just misleads/confuses people now. 
```
// We will act as if they clicked on the first verb in that category.
```

Maybe just remove it or update it in this pr?